demo/config/targets/INST/cmd_tlm/inst_cmds.txt

```diff
@@ -34,12 +34,12 @@ COMMAND INST SETPARAMS BIG_ENDIAN "Sets numbered parameters"
 
 COMMAND INST ASCIICMD BIG_ENDIAN "Enumerated ASCII command"
   <%= render "_ccsds_cmd.txt", locals: {id: 5} %>
-  PARAMETER    STRING         64  2048 STRING "NOOP"    "Enumerated string parameter"
+  APPEND_PARAMETER STRING 2048 STRING "NOOP" "Enumerated string parameter"
     STATE "NOOP" "NOOP"
     STATE "ARM LASER" "ARM LASER" HAZARDOUS "Arming the laser poses an eye safety hazard."
     STATE "FIRE LASER" "FIRE LASER" HAZARDOUS "WARNING Laser will be fired!"
   APPEND_PARAMETER BINARY 32 STRING 0xDEADBEEF "Binary string"
-  APPEND_PARAMETER STRING 80 STRING "0xDEADBEEF" "ASCII string"
+  APPEND_PARAMETER ASCII 80 STRING "0xDEADBEEF" "ASCII string"
```

Comment:
While looking at the warnings I realized this was overlapping the old STRING so I changed it